src/content/translations/zh/developers/docs/nodes-and-clients/index.md

```diff
@@ -41,8 +41,8 @@ sidebarDepth: 2
 
 ### 归档节点 {#archive-node}
 
-- 存储保留在完整节点中的所有内容，并建立历史状态档案。 如果您想查询区块 #4,000,000 的帐户余额等内容，或者简单可靠地[测试自己的交易集，而不使用 OpenEthereum 挖矿](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany)，则需要这样做。
-- 这些数据以太字节为单位，这使存档节点对普通用户的吸引力降低，但对于诸如区块浏览器、钱包供应商和链分析之类的服务而言却很方便。
+- 存储保留在全节点中的所有内容，并建立一个历史状态的归档。 如果您想查询区块 #4,000,000 的帐户余额等内容，或者简单可靠地[使用OpenEthereum在不挖坑的情况下测试自己的交易集](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany)，则需要这样做。
```

Comment:
存储所有保存在全节点中的内容，并建立一个历史状态的归档。 如果您想查询区块 #4,000,000 的帐户余额等内容，或者简单可靠地[使用 OpenEthereum 在不挖矿的情况下测试自己的交易集](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany)，就需要这样做。

(might be better to leave a space between EN / CN? Also it should be "挖矿“ rather than “挖坑”.) 😄

Reply:
Nice!